src/components/DocumentView/OpenAPI/style.css

```diff
@@ -172,6 +172,10 @@
     @apply prose-sm;
 }
 
+.openapi-schema-example {
+    @apply mt-2 text-base text-dark/10 dark:text-light/10;
```

Comment:
Why are we using a larger text font than the description?

It doesn't look good in the screenshot.

Reply:
fixed. see the screenshot above